composer.json

```diff
@@ -9,7 +9,7 @@
 	},
 	"require": {
 		"php": "^7.0",
-		"squizlabs/php_codesniffer": "~2.8.1"
+		"squizlabs/php_codesniffer": "dev-master"
```

Comment:
Hey, I also need this PR to be merged.

But my custom sniff repository currently depends on `"squizlabs/php_codesniffer": "~3.0.0"` .

So I cannot require this branch(also future merged branch).

I suggest the dependency version `~3.0.0` or `^3.0` will be better than `dev-master` .

(I say Sorry when you supposed to change this version when PHPCS3.0.1 released.)

Thanks.